### ISSUE TITLE
Why does the order of the head attributes change in the generated new page？

### ISSUE THREAD

**Issue by jiangzhonglian** [getpelican/pelican#3130]
The system page is normal, but when I compiled my own MD file into a page, it changed！

```html
<head>
    <meta name="generator" content="Pelican" />
    <script async src="xxx" crossorigin="anonymous"></script>
</head>
```

I am using the command: pelican content，and It will be changed to:

```
<head>
    <meta content="Pelican" name="generator"/>
    <script async="" crossorigin="anonymous" src="xxx"></script>
</head>
```

Why does the order of attributes change, and how can we ensure that it stays the same?

**Comment by avaris** [comment]
I don't quite understand. 

First, attribute order in HTML tags does not matter. So... why do you care?

Second, are you placing `<head>` inside a markdown file? That won't normally work. HTML generated with content files are supposed to be placed inside `<body>` and `<head>` is pointless there... Unless theme does something unconventional.

**Comment by jiangzhonglian** [comment]
In the script, async becomes async=""

<img width="681" alt="image" src="https://github.com/getpelican/pelican/assets/9199175/da9709c2-aa81-4d57-9212-c45e00e9b44b">

**Comment by avaris** [comment]
Yes and that does not matter as well. `async` is a boolean attribute. If it is present, it will be enabled. It doesn't matter if it has a value or what the value is.

**Comment by jiangzhonglian** [comment]
Although you provided an answer, it's not what I don't want. I just want to add some code into the "head" section, and ensure that the code won't be modified during the website compilation.

**Comment by justinmayer** [comment]
@jiangzhonglian: When you have more information to provide, post a comment here so we can consider re-opening this issue. Cheers!
